components/form/RadioButton.vue

```diff
@@ -138,6 +138,10 @@ export default {
   border-radius: var(--border-radius);
   padding-bottom: 5px;
 
+  &.disabled {
+    cursor: default
```

Comment:
Is this correct? Based on the naming I would think this should be 'not-allowed'

Reply:
Good point